### ISSUE TITLE
Protective ("magic") texts as additions

### ISSUE THREAD

**Issue by thea-m** [BetaMasaheft/Documentation#923]
Split from #895 

From @DenisNosnitsin1970 's e-mail, the difference between the addition types "MagicText" and "ProtectivePrayer" was clarified: "ProtectivePrayer" is the preferable term, "magic" was dropped at a later stage of Ethio-Spare. We should therefore delete the type MagicText and replace it by ProtectivePrayer wherever it has been used (I can do that). 
I will also add an addition type "Recipe" for protective recipes.
A further question is how to deal with protective texts of which we are not sure whether they are prayers.  We could create another type for them, but since these types of additions are often very difficult to distinguish.  it could make sense to replace "ProtectivePrayer" by the more general "ProtectiveText"? What do you think?

(The keyword "magic" should probably also be replaced by "ProtectiveWork" or something similar - what would you propose?)

**Comment by DariaElagina** [comment]
I think that Gidena discussed the problematic of the word 'magic' in his dissertation. I could consult it for his proposed solutions.

**Comment by thea-m** [comment]
@DariaElagina Thank you! Please let us know here.

**Comment by DariaElagina** [comment]
Gidena replaces the term 'magic' with 'abǝnnät'. (Actually, not all 'magic' texts are protective in function.) I am not sure whether this would be the best solution, but some texts containing the word magic in the description are apparently those to which Gidena refers as abǝnnät. 
 I quote here some excerpts from his thesis:
"Replacing the term ‘magic’ with abǝnnät is not simply based on hints we find in the works of 
the scholars Burtea related in the above discussion. To put is on a solid ground, our choice of
the  term አብነት abǝnnät is  substantiated  by the  definitions of  this  very  term  rendered  in 
different  dictionaries,  catalogue  information,  interviews  carried  out  (oral  account)  and 
internal textual evidence from within the texts."
"Eventually, one can see the recurrence of the meanings ‘model and pattern’ and the relation 
of the term to the conceptions of healing and superstition. Hence, the term abǝnnät can be 
applied to denote a textual unit the contents of which are phylacteries, herbal remedies and 
divinatory  texts  with  a  specific  application  formula.  Eventually  it  must  be  noted  that  the 
abǝnnät is a single whole with a specific function."
Attached is his thesis. 
[Dissertation of Gidena.pdf](https://github.com/BetaMasaheft/Documentation/files/2141365/Dissertation.of.Gidena.pdf)

**Comment by eu-genia** [comment]
I am not sure the term Gidena proposes is acceptable; this is something he more or less invented, but we should also take care of being easily understood. While "magic" is indeed something that is considered politically incorrect (even if widely used), there are other more neutral terms like e.g. "apotropaic".

**Comment by DenisNosnitsin1970** [comment]
Abǝnnät is the Amharic term primarily used for "recipe", in that context, or also "example",  "guidelines" for magic rituals. It was never clear for me why he decided to apply this term to all "magic texts". But it there would have been such a justification, "abǝnnät-prayer", "abǝnnät-text" are very problematic as they are completly non-transparent. This is a complex term in itself; in addition to that, abǝnnät tǝmhǝrt bet is used to refer to the traditional school.

**Comment by eu-genia** [comment]
The widest used English language term for all related traditions (and we want to be understood also by those who do e g Syriac or Coptic studies) is still "magical (not magic) texts". That's the keyword users shall most frequently be looking for. So if we replace it with anything possibly a mapping should be added to the taxonomy and the search for magic be still made possible, even if the keyword as such is replaced with something else.

@abausi may have some ideas too.

**Comment by PietroLiuzzo** [comment]
to change the label of a keyword, WITHOUT changing the value in the taxonomy, schema and data, it is enough to change the title of the Authority file

**Comment by thea-m** [comment]
@eu-genia , I agree that user friendliness is very important, but I nevertheless don't think that we should overeagerly use problematic terms (that are controversial for a reason). If someone scrolls through the list looking for "magic" and doesn't find it, they will probably get the idea to click on protective/apotropaic?

**Comment by thea-m** [comment]
The consensus seems to be not to use ʾabənnat. 
I would be in favour of the terms ProtectivePrayer, ProtectiveText and Recipe, adding a definition to the records which clarifies that our use of these terms includes also strictly speaking non-protective texts (but I also understand the arguments in favour of "Magic(al)").
@abausi

**Comment by abausi** [comment]
Se shall discuss this. We see advantages and disadvantages. Not always 
and not necessarily must we conform to current definitions (see for 
example the discussion on colophons, and titles). "Texts of ritual 
power" was recently proposed; "abǝnnat" has also its reasons.

Il 02.07.2018 11:30, thea-m ha scritto:

**Comment by PietroLiuzzo** [comment]
Please update the schema and the guidelines to contribute to close this issue, we can discuss the proposal in the next meeting

**Comment by eu-genia** [comment]
Anything that needs to be done here? Can probably be closed, @thea-m ?
